api/filters/namespace/namespace.go

```diff
@@ -19,9 +20,28 @@ type Filter struct {
 	// FsSlice contains the FieldSpecs to locate the namespace field
 	FsSlice types.FsSlice `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 
+	// SkipExisting means only blank namespace fields will be set
+	SkipExisting bool `json:"skipExisting" yaml:"skipExisting"`
```

Comment:
This config can be helpful. One case users' asking for is that they want to only replace the namespace if it's empty string (the actual "default" ns). Will this config treat empty string namespace as "existing"?

Reply:
No, empty string does not count as existing. So `skipExisting: true` will replace the namespace field if its value is empty or if it is not present at all.